group-dep-types.json

```diff
@@ -1,35 +1,32 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "platformAutomerge": true,
-  "rebaseWhen": "never",
+  "automergeType": "pr",
+  "automergeStrategy": "squash",
   "packageRules": [
     {
-      "description": "Automerge non-major NPM dev dependencies off business hours",
+      "description": "Automerge non-major NPM dev dependencies",
       "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
       "matchDatasources": ["npm"],
       "matchDepTypes": ["devDependencies"],
       "schedule": [
-        "after 10pm every weekday",
-        "every weekend",
-        "before 5am every weekday"
+        "after 8pm every weekday",
+        "before 8am every weekday",
```

Comment:
I believe this is in PST so this before will actually be within the work day for mountain and EST

Reply:
Yep. I can revert but I'm trying to increase the available window where possible.